src/main/java/com/google/sps/utils/data/Animal.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.HashSet;
+
+public class Animal {
+    
+    // Cross-referenced
+    private String common_name;
+    private String binomial_name;
+
+    // Provided by Wikipedia
+    private int population;
+    private String status;
+
+    // Provided by IUCN API
+    private int id;
+    private String kingdom;
+    private String phylum;
+    private String class;
+    private String order;
+    private String family;
+    private String genus;
+    private String population_trend;
+    private boolean marine_system;
+    private boolean freshwater_system;
+    private boolean terrestrial_system;
+    private String taxonomic_notes;
+    private Collection<String> countries;
+    private Collection<AnimalHistory> history_set;
+    private String citation_link;
```

Comment:
It might be nice to have a list of citation links for each animal since we'd also need to cite wikipedia.